partner_address_hide_pdf_fields/tests/fake_models.py

```diff
@@ -0,0 +1,20 @@
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _name = "res.partner"
+    _inherit = "res.partner"
+    _description = "Fake ResPartner for Testing"
+
+    def _hided_in_pdf_address_fields(self):
```

Comment:
Please provide a comment to let us know what extra functions you do in this Fake model.